modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -106,7 +112,6 @@
             :is-modified="isModified"
             :module-labels="moduleLabels"
             @back="updateEditing(null)"
-            @saved="updateMedia"
```

Comment:
Why?

Reply:
Get media was done twice with the `saved` event and `content-changed` one.